app/views/layouts/mailer.html.erb

```diff
@@ -25,6 +25,49 @@
         font-weight: normal;
         margin-bottom: 2.5rem;
       }
+      .email-contents button {
+          background-color: #121111;
+          border-bottom: #121111 2px solid;
+          border-bottom-left-radius: 10px;
+          border-bottom-right-radius: 10px;
+          border-image-outset: 0;
```

Comment:
I don't think we need any of these either, the button looks the same when I remove it:
```
- border-image-outset: 0;
- border-image-repeat: stretch;
- border-image-slice: 100%;
- border-image-source: none;
- border-image-width: 1;

```